src/packer.pkr.hcl

```diff
@@ -142,13 +151,185 @@ build {
+      "src/powershell/enable-rdp.ps1",
+      "src/powershell/initialize-volume.ps1",
+      "src/powershell/install-chocolatey.ps1"
+    ]
+  }
+
+  provisioner "powershell" {
+    # Create "packages" directory before uploading them in the next provisioner.
+    inline = ["mkdir D:\\packages"]
+  }
+
+  provisioner "file" {
+    # Upload package lists to the "packages" directory.
+    source      = "src/packages/"
+    destination = "D:\\packages"
+  }
+
+  provisioner "powershell" {
+    # Install general packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=general"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install Docker packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=docker"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install evasion packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=evasion"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install exploitation packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=exploitation"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install information gathering packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=information-gathering"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install Kali packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=kali"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install networking packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=networking"]
```

Comment:
```suggestion
    environment_vars = ["CATEGORY=networking"]
    script           = "src/powershell/install-category.ps1"
```

Reply:
Resolved in https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/504430a928e07070bb27846baaea48cf51db677a.